lib/activerecord-import/import.rb

```diff
@@ -961,7 +961,7 @@ def values_sql_for_columns_and_attributes(columns, array_of_attributes) # :nodoc
           if val.nil? && Array(primary_key).first == column.name && !sequence_name.blank?
             connection_memo.next_value_for_sequence(sequence_name)
           elsif val.respond_to?(:to_sql)
-            "(#{val.to_sql})"
+            "('#{val.to_sql}')"
```

Comment:
This isn't going to work. The `to_sql` call can return sub queries.

Reply:
i see..